homeassistant/components/tractive/sensor.py

```diff
@@ -0,0 +1,184 @@
+"""Support for Tractive sensors."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from homeassistant.components.sensor import SensorEntity, SensorEntityDescription
+from homeassistant.const import (
+    ATTR_BATTERY_LEVEL,
+    DEVICE_CLASS_BATTERY,
+    PERCENTAGE,
+    TIME_MINUTES,
+)
+from homeassistant.core import callback
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+
+from .const import (
+    ATTR_DAILY_GOAL,
+    ATTR_MINUTES_ACTIVE,
+    DOMAIN,
+    SERVER_UNAVAILABLE,
+    TRACKER_ACTIVITY_STATUS_UPDATED,
+    TRACKER_HARDWARE_STATUS_UPDATED,
+)
+from .entity import TractiveEntity
+
+
+@dataclass
+class TractiveSensorEntityDescription(SensorEntityDescription):
+    """Class describing Tractive sensor entities."""
+
+    event_type: str | None = None
+    attributes: tuple = ()
+
+
+ATTR_ACTIVITY = "activity"
```

Comment:
Does it make sense to move these constants to const.py?

Reply:
It's only used here?